data_structures/binary_tree/basic_binary_tree.py

```diff
@@ -4,13 +4,13 @@ class Node:
     and left, right pointers.
     """
 
-    def __init__(self, data):
-        self.data = data
-        self.left = None
-        self.right = None
+    def __init__(self, data: int) -> None:
+        self.data: int = data
```

Comment:
```suggestion
        self.data = data
```
Not required because mypy can intuit based on the context that we are assigning an int value.
Let's focus our type hints on function parameters and return values unless mypy says it needs a type hint.